### PR TITLE
Deprecate visibility of SizeCompareOps#it

### PR DESCRIPTION
**Pull request by NthPortal**
oops

### CODE REVIEW

**Comment by NthPortal** [comment]
it would be nice to have a `@deprecatedVisibility` or `@deprecatedAccess` annotation for this sort of thing

File: src/library/scala/collection/Iterable.scala [hunk 1]
```diff
@@ -850,19 +850,22 @@ object IterableOps {
     * These operations are implemented in terms of
     * [[scala.collection.IterableOps.sizeCompare(Int) `sizeCompare(Int)`]].
     */
-  final class SizeCompareOps private[collection](val it: IterableOps[_, AnyConstr, _]) extends AnyVal {
+  final class SizeCompareOps private[collection](private val coll: IterableOps[_, AnyConstr, _]) extends AnyVal {
```
Comment (som-snytt) [inline src/library/scala/collection/Iterable.scala:853]: just `(@deprecated("you can't see me blushing","") val it`. FSR `it` changes `its` name from `it` to `it$extension`.

Comment (NthPortal) [reply src/library/scala/collection/Iterable.scala:853]: well. that's frustrating. unfortunately, doing what you suggest means there are six unavoidable deprecation warnings for its use within the value class

**Comment by NthPortal** [comment]
I guess I'll close this and wait for someone to implement the annotation I mentioned

**Pull request closed without merging**